dashboard/app/models/script.rb

```diff
@@ -1086,8 +1086,7 @@ def self.add_unit(options, raw_lesson_groups, new_suffix: nil, editor_experiment
 
       unit.generate_plc_objects
 
-      CourseOffering.add_course_offering(unit)
-
+      CourseOffering.add_course_offering(unit) if unit.is_course
```

Comment:
this looks like a good improvement -- but it also looks like we also do this here: https://github.com/code-dot-org/code-dot-org/blob/a4ebae23f5537c0401a54d31f68c60bed817496b/dashboard/lib/services/script_seed.rb#L203 any idea whether one of these can/should be removed?

Reply:
to answer my own question, I guess what I'm seeing is:
1. the call in this file is needed for unmigrated units
2. the call in script_seed.rb (according to the comment) needs to be done at a specific time so that resources and vocab can be seeded. 

since the method looks safe to call twice, and since unmigrated scripts are scheduled to go away soon, this seems fine to leave as you have it.

Reply:
oh interesting...didn't realize it was duplicated. But, like you pointed out below, this is safe to call multiple times